src/controllers/QueryController.ts

```diff
@@ -596,12 +596,14 @@ export class QueryController extends RootComponent {
       'aq',
       'cq',
       'dq',
+      'lq',
       'searchHub',
       'tab',
       'pipeline',
       'sortCriteria',
       'recommendation',
-      'commerce'
+      'commerce',
+      'facets'
```

Comment:
Hmm, could there have been a way to prevent forgetting these keys?

Reply:
They were caught by end to end tests 🤷